apps/tailwind-components/components/form/Fields.vue

```diff
@@ -127,23 +131,6 @@ function checkVisibleExpression(column: IColumn) {
   );
 }
 
-function onUpdate(column: IColumn, $event: columnValue) {
-  dataMap[column.id] = $event;
```

Comment:
todo, find out where this logic went

Reply:
like a 'watch' of some sort?

Reply:
we need it so that if somebody fixes an error it immediately removes the error message. See example here: I typed -11000 so that gave an error onblur. But then I went back and removed the - but error stayed. onUpdate aims to fix that.
<img width="1059" alt="image" src="https://github.com/user-attachments/assets/600796fa-1819-427f-9387-38eabd1b0450" />